tests/cache/tests.py

```diff
@@ -1256,6 +1256,24 @@ def test_close(self):
         finally:
             signals.request_finished.connect(close_old_connections)
 
+    def test_set_many_returns_failed_keys(self):
+        k1 = "key1"
+        k2 = "key2"
+        value = 'a' * 1024 * 1024 * 2  # invalid value, too big for memcached
+        try:
+            res = cache.set_many({k1: 'x', k2: value})
+            self.assertTrue(res)
+            self.assertIn(k2, res[0])
+        finally:
+            # memcached seems puzzled by the attempt to insert a 2MB
```

Comment:
This doesn't look too nice. I'm not sure if mocking as suggested on the original PR would be better.